tests/topotests/ospf6_loopback_cost/test_ospf6_loopback_cost.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+# SPDX-License-Identifier: ISC
+
+# Copyright (c) 2023 by
+# Donatas Abraitis <donatas@opensourcerouting.org>
+#
+
+"""
+Test if OSPFv3 loopback interfaces get a cost of 0.
+
+https://www.rfc-editor.org/rfc/rfc5340.html#page-37:
+
+If the interface type is point-
```

Comment:
Cosmetic: You might want to add the same amount of spaces here as in subsequent lines of the RFC quote.

Reply:
Fixed.